instrumentation/net/http/otelhttp/config.go

```diff
@@ -194,3 +198,11 @@ func WithServerName(server string) Option {
 		c.ServerName = server
 	})
 }
+
+// WithReqAttributesMapper returns an Option to set a function that maps an HTTP request to a slice of attribute.KeyValue.
+// These attributes will be included in metrics for every request.
+func WithReqAttributesMapper(reqAttributesMapper func(r *http.Request) []attribute.KeyValue) Option {
```

Comment:
This method name is still not metric-specific, when the data is. How about `WithMetricAttributesFn`?

Reply:
Yea, is more appropriate, done ✅